portfolio/src/main/java/com/google/sps/servlets/comment.java

```diff
@@ -0,0 +1,12 @@
+package com.google.sps.servlets;
+
+public final class comment {
```

Comment:
Perhaps rename this to be "Comment" instead.

Reply:
yes, you are right! my bad with the style...